tests/pkg_micro-ecc/tests/01-run.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import math
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+
+def testfunc(child):
+    child.expect_exact('micro-ecc compiled!')
+    child.expect_exact('Testing 16 random private key pairs and signature '
+                       'without using HWRNG')
+    child.expect_exact('................ done with 0 error(s)')
```

Comment:
This line takes few second more than 30 seconds on `samr21-xpro`. So please set a longer timeout for this line, 60 seconds would give some margin.

Reply:
that's one of the reason I'm against such rigorous string checking, but would rather look for a final `[SUCCESS]` with sufficient timeout and assume failure if that doesn't show up.

I guess most of our test script will fail when run on real hardware, just because of timeouts e.g. due to slow serial connect (or getting ready)  - not because there is an error.

Reply:
@cladmi, do you mean setting a global timeout to the test runner ?

Reply:
ok, I think I have it, setting `timeout=60` passes for me on a nucleo board. @smlng it's not a problem of testing per-line but of the total time the test take to complete. `testrunner` has a timeout option for that.

Reply:
Putting a timeout on this line or globally. (you currently removed the timeout https://github.com/RIOT-OS/RIOT/pull/7992/commits/048ca3f1dfc36485955762856aae34c66aaacca6)

@smlng tests should work on real hardware. Getting ready is taken care of, there is now a sleep after make term, and slow serial connection is now also by removing all serial timing.
All the existing python tests and newly added tests work on native and samr21-xpro, if the board is supported.

Reply:
> you currently removed the timeout 048ca3f

Yes, sorry, I don't what I had in mind when I did that... reverted. Now there's a 60s timeout for both micro-ecc package tests